content/posts/blog/2021-01-24_blog_MessagerieInstantanee-Ilnesagitpasdelapplication

```diff
@@ -0,0 +1,59 @@
+
+### Souveraineté de la communication
+
+Comme vous vous en doutez, je vais plaider en faveur de l'XMPP. Je pense que le premier choix à faire n'est pas de savoir quel messager utiliser, mais quelle technologie logicielle sous-jacente. Il faut d'abord réfléchir au choix de la technologie, avant de passer d'une recommandation à l'autre.
+
+> XMPP est un protocole ouvert, comme HTTP l'est pour le web. XMPP est un protocole ouvert, comme HTTP l'est pour le web. Peu importe à quoi ressemble votre site web, tout le monde peut interagir avec lui. Ce principe est l'idée qui sous-tend XMPP, mais pour la messagerie instantanée.
+
+À la Fondation des normes XMPP, et avec de nombreux autres acteurs impliqués dans cette technologie ouverte, nous pensons que XMPP est un excellent choix en tant que technologie de communication, et pas seulement en ce qui concerne la confidentialité des données. XMPP existe depuis plus de vingt ans, et une grande expérience pratique a été acquise au cours de cette période. Elle comprend de nombreux indicateurs clés pour le choix de la technologie et la souveraineté habilitante de la communication, comme elle le soutient :
+
+- Décentralisation des services de communication (fédération)
+- Normalisation et extensibilité de la technologie
+- Interopérabilité
+- L'innovation et l'utilisation du développement ouvert
+- Respect de la vie privée et contrôle, notamment par l'utilisation d'un cryptage de bout en bout
+
+> XMPP fournit des informations définies sur la manière de gérer la communication dans un réseau. L'utilisateur décide quel logiciel de messagerie convient ou où les données doivent être stockées.
+
+Contrairement à la situation actuelle où d'autres personnes vous suggèrent d'utiliser leur messager préféré, dans l'univers XMPP vous avez le libre choix de nombreux messagers - choisissez celui que vous aimez, pas celui que les autres aiment 
+jusqu'au prochain engouement pour l'internet. La différence est que, quelle que soit votre application de messagerie ou celle qu'utilisent vos amis, vous avez toujours le confort d'être sur le même réseau avec tous vos contacts, indépendamment de votre choix aujourd'hui. Je pense que c'est une vraie solution.
+
+### Il ne s'agit pas de l'application, mais de la technologie
+
+Réfléchissez à la technologie que vous comptez utiliser, puis décidez quel messager vous convient ou convient à votre environnement. XMPP et sa communauté ont accumulé une expérience variée, qui a été 
+utilisé dans de nombreux logiciels. Nous pensons que c'est une bonne solution pour la plupart des gens et des organisations, et pour l'Internet dans son ensemble - et il est ouvert à tous pour faire des innovations pour l'avenir.
```

Comment:
utilisé → utilisée (talking about the experience)

“et il est ouvert à tous pour faire des innovations pour l'avenir” → “et il reste ouvert à des innovations futures" ?